### PR TITLE
base image version bump

### PR DESCRIPTION
**Pull request by vsamidurai**
Dockerfile base image version bump to fix multiple CVE.

https://github.com/gliderlabs/docker-alpine/issues/430

#567

### CODE REVIEW

**Review by ldelossa — DISMISSED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -22,7 +22,12 @@ RUN export CLAIR_VERSION=$(git describe --tag --always --dirty) && \
 FROM alpine:3.10
 COPY --from=build /go/src/github.com/coreos/clair/clair /clair
 RUN apk add --no-cache git rpm xz ca-certificates dumb-init
-USER nobody
+
+
+# change ownership of ssl directory to allow custom cert in OpenShift
+RUN chgrp -R 0 /etc/ssl/certs && \
```
Comment (ldelossa) [inline Dockerfile:27]: @vsamidurai can you explain this a bit more? Are you expecting to use a volume to mount certs to this path?

Comment (vsamidurai) [reply Dockerfile:27]: @ldelossa  that was added part of another change #755 . I just updated PR with conflict.

**Review by KeyboardNerd — DISMISSED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -19,10 +19,11 @@ WORKDIR /go/src/github.com/coreos/clair/
 RUN export CLAIR_VERSION=$(git describe --tag --always --dirty) && \
 	go build -ldflags "-X github.com/coreos/clair/pkg/version.Version=$CLAIR_VERSION" github.com/coreos/clair/cmd/clair
 
-FROM alpine:3.8
+FROM alpine:3.10
 COPY --from=build /go/src/github.com/coreos/clair/clair /clair
 RUN apk add --no-cache git rpm xz ca-certificates dumb-init
 
+
```
Comment (KeyboardNerd) [inline Dockerfile]: nit: extra space

Comment (vsamidurai) [reply Dockerfile]: thank you.. removed extra space

**Review by KeyboardNerd — APPROVED** [review]

**Pull request merged**